From: Debian Science Maintainers
 <debian-science-maintainers@lists.alioth.debian.org>
Date: Sat, 5 Oct 2019 11:45:42 +0000
Subject: No spninx issues

Remove dependency from sphinx-issues (not packaged in debian)
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 0c978ac..c42651f 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -47,7 +47,7 @@ extensions = [
     'sphinx.ext.autosummary',
     'sphinx.ext.viewcode',
     'numpydoc',
-    'sphinx_issues',
+    # 'sphinx_issues',
 ]
 
 numpydoc_show_class_members = False
